net/acme-common/files/acme.sh

```diff
@@ -117,7 +117,6 @@ usage() {
 Usage: acme <command> [arguments]
 Commands:
 	get                issue or renew certificates
-	cert <domain>      show certificate matching domain
```

Comment:
This change is unrelated to the commit message; please split it into a separate commit...